### ISSUE TITLE
GCW has NA Instrument names

### ISSUE THREAD

**Issue by stephpenn1** [COMPASS-DOE/data-workflows#205]
![image](https://github.com/user-attachments/assets/c5c18cee-4842-4730-9c4e-a1b41ae03dde)

![image](https://github.com/user-attachments/assets/e21e112e-5169-4237-9b00-f2ceea12f87f)

**Comment by stephpenn1** [comment]
@bpbond also just realized my x axis is incorrectly ordered, any thoughts on how to fix this?

```
 mutate(
           ts_str = format(date, "%b-%Y"),
           ts_fct = factor(ts_str, levels = unique(ts_str)))
```

**Comment by bpbond** [comment]
```r
arrange(date) %>%
mutate(
           ts_str = format(date, "%b-%Y"),
           ts_fct = factor(ts_str, levels = unique(ts_str)))
```

?

**Comment by stephpenn1** [comment]
Hmmm no, the dataset is arranged correctly with this but is graphing the same. I just pushed up `availability_graph.R` to `heatmap` branch if you want to take a look at the entire code

**Comment by bpbond** [comment]
See PR! I didn't test it on the full dataset though—please lmk if it doesn't fix things

**Comment by bpbond** [comment]
Fixed in #206
